config/crds/kyverno.io_generaterequests.yaml

```diff
@@ -182,9 +182,3 @@ spec:
     storage: true
     subresources:
       status: {}
-status:
```

Comment:
Was this removed by a newer version of Kustomize?

Reply:
Yes, new version of controller-gen.